crates/oxc_linter/src/snapshots/no_named_as_default.snap

```diff
@@ -0,0 +1,34 @@
+---
+source: crates/oxc_linter/src/tester.rs
+assertion_line: 143
+expression: no_named_as_default
+---
+  ⚠ eslint-plugin-import(no-named-as-default): Using exported name "foo" as identifier for default export.
```

Comment:
Do we need to say what the export name is from the other side? The current messages doesn't say about why this is bad.

Reply:
I've used [same](https://github.com/import-js/eslint-plugin-import/blob/7a21f7e10f18c04473faadca94928af6b8e28009/tests/src/rules/no-named-as-default.js#L71C1-L72C1) error message as in eslint plugin. What do you want to add in it?

Reply:
Maybe something like: `Module './bar' has named export 'foo'. Naming default import as 'foo' can be confusing`

Reply:
@XantreGodlike sounds good to me.

Reply:
@Boshen fixed